Client/Frontend/Fakespot/Views/FakespotHighlightsCardView.swift

```diff
@@ -86,6 +95,12 @@ class FakespotHighlightsCardView: UIView, ThemeApplicable {
     private var viewModel: FakespotHighlightsCardViewModel?
     private var isShowingPreview = true
 
+    private var safeAreaEdgeInsets: UIEdgeInsets {
+        guard let firstScene = UIApplication.shared.connectedScenes.first as? UIWindowScene else { return UIEdgeInsets() }
```

Comment:
Can we move this as a  `UIApplication.shared` extension? Doesn't seem to be related to this view.

Reply:
Yes, sure.